userguide/kaui/includes/overdue.adoc

```diff
@@ -0,0 +1,188 @@
+:imagesdir: C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\kaui
+
+=== Overview
+
+Kaui lets you configure basic settings for how the system behaves when an account is overdue. In Kaui, this is a subset of the full overdue configuration settings and is defined at the tenant level. For more information, see the https://docs.killbill.io/latest/overdue.html[_Overdue System Guide_].
+
+[NOTE]
+*Note:* You can also configure overdue actions for the _global_ Kill Bill system (as opposed to configuring at the tenant level). For more information, see the https://docs.killbill.io/latest/overdue.html[_Overdue System_] documentation.
+
+See the following sections to configure overdue (dunning) flows for a tenant:
+
+[cols="1,1"]
+[frame=none]
+[grid=none]
+|===
+
+a| * <<View Overdue Configuration,View overdue configuration>>
+* <<Configure Overdue Flows,Configure overdue flows>>
+* <<Upload an XML Overdue Configuration File,Upload an XML overdue configuration file>>
+
+a| * <<View Overdue Configuration as XML,View overdue configuration as XML>>
+* <<Delete Overdue Flows,Delete overdue flows>>
+
+|===
+
+You cannot edit a raw overdue configuration XML file in Kaui.
+
+==== Additional Overdue Resources
+
+* https://docs.killbill.io/latest/userguide_subscription.html#components-overdue["Overdue"] section in the _Subscription Guide_.
+* https://docs.killbill.io/latest/overdue.html[_Overdue System Guide_]
+
+=== View Overdue Configuration
+
+If any overdue (dunning) actions are configured, you can see them on the Overdue Show tab at the bottom of the Tenant Configuration page. To get there:
+
+. Click on your username and tenant name in the upper right corner:
++
+image::Users-SelectTenantName.png[width=650,align="center"]
++
+Kaui displays the Tenant Configuration page.
++
+. Scroll down and click on the Overdue Show tab:
++
+image::OverdueTab.png[width=850,align="center"]
+
+=== Configure Overdue Flows
+
+
+The following steps explain how to set up overdue flows (i.e., actions) for the tenant.
+
+. <<View Overdue Configuration,Go to the Overdue Show tab>> on the Tenant Configuration page.
++
+image::OverdueTab.png[width=850,align="center"]
++
+. Click the plus sign ( image:i_PlusGreen.png[] ) next to *Existing Overdue Config*.
++
+Kaui opens the Overdue Configuration screen:
++
+image::OverdueConfig-Empty.png[width=850,align="center"]
++
+. Click *+ New Overdue States*. Kaui opens the fields for editing:
++
+image::OverdueConfig-BlankFields.png[width=850,align="center"]
++
+. Fill in the fields for the current row. For field information, see the <<Overdue Configuration Field Descriptions>> table.
++
+. To add another overdue configuration row, click *+ new overdue states*.
++
+image::Overdue-Select-NewOverdueState.png[width=450,align="center"]
++
+. When you are done adding configuration rows, click the *Save* button. Kaui displays the new row(s) on the Overdue Show tab:
++
+I NEED A SCREENSHOT OF THIS
+
```

Comment:
You can use the attached screenshot. I've also observed a few scenarios when it is not possible to save the overdue configuration. I've filed[ this issue](https://github.com/killbill/technical-support/issues/48) to track it. 
![save-overdue-state](https://user-images.githubusercontent.com/85998496/164414152-fc32e2f1-0213-4966-8fb6-838e084c340c.png)

Reply:
@reshmabidikar - The screenshot above shows the Catalog tab, not the Overdue tab.

Reply:
@Flowwriter, when the "Save" button is clicked, the browser displays the "Catalog" tab with a message at the top of the screen that says "Overdue Config Successfully Added". I agree with you, Kaui should ideally display the "Overdue" tab.

Reply:
I decided to show the Overdue tab but just mention the success message. This is page 3.